packages/smooth_app/lib/widgets/attribute_helper.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:openfoodfacts/personalized_search/preference_importance.dart';
+
+// Common values and methods for attribute display
```

Comment:
The functions here aren’t in an own class so I can imagine it being confusing to call getIcon or anything else here and then get asked to provide an importanceId. So I'd recommend renaming all the functions here to have attribute in their name.

Reply:
The best practice in flutter/dart in those cases is to name the file - `import '...' as myprefix;`
Then the call would be `myprefix.getIcon(...)`.